met-api/tests/unit/api/test_engagement.py

```diff
@@ -49,3 +49,23 @@ def test_get_engagements(client, jwt, session, engagement_info):  # pylint:disab
 
     assert created_eng.get('result').get('name') == rv.json.get('result').get('name')
     assert created_eng.get('result').get('content') == rv.json.get('result').get('content')
+
+
+@pytest.mark.parametrize('engagement_info', [TestEngagementInfo.engagement1])
+def test_patch_engagement(client, jwt, session, engagement_info):  # pylint:disable=unused-argument
+    """Assert that an survey can be POSTed."""
+    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.no_role)
+    engagement = factory_engagement_model()
+    engagement_id = str(engagement.id)
+    new_engagement_name = 'new_engagement_name'
+    rv = client.patch('/api/engagements/', data=json.dumps({'id': engagement_id, 'name': new_engagement_name}),
+                      headers=headers, content_type='application/json')
+
+    assert rv.status_code == 200
```

Comment:
Does that patch returns something?May be assert that as well.Will be easier in the future incase if we are changing the patch response to something else

Reply:
done